.env

```diff
@@ -0,0 +1,6 @@
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_SECRET=CHANGE_ME_TO_SOMETHING_SECRET_AND_RANDOM
```

Comment:
being here it means dev setup will use the env var, so this needs to be updated.
maybe I should remove it? kept it because it impacts only dev setup, not prod ones